symbols/arm9.yml

```diff
@@ -1010,6 +1010,38 @@ arm9:
         r0: [output] pointer to an IO struct {ptr, len}
         r1: file path string pointer
         r2: flags
+    - name: UpdateFadeStatus
+      address:
+        EU: 0x200BA18
+      description: |-
+        Updates the given screen_fade struct to initiate a fade for example.
+        
+        In addition to initiating a fade this is called when a fade out is complete
+        to set a flag for that in the struct.
```

Comment:
YAML files don't have a character limit, so line breaks are usually not added here (except at the end of a paragraph). Not sure if it matters though.

Reply:
Eh I usually don't add line breaks, but I don't enforce a standard. Others like Hecka usually add line breaks.